opentracing-specialagent/src/test/java/io/opentracing/contrib/specialagent/AgentRunner.java

```diff
@@ -434,6 +434,9 @@ public AgentRunner(final Class<?> testClass) throws InitializationError, Interru
       }
     }
 
+    System.setProperty("sa.config.instrumentation.plugin.method",
```

Comment:
@code98, please remove this from `AgentRunner`. The `AgentRunner` cannot know anything about any specific plugins.

Reply:
Because 'MethodAgentRule' class has 'System.getProperty' code to get environment variables. Because @Before and @BeforeClass have lower priority when unit testing, setting 'System.setProperty' cannot run successfully. Can only be added in 'AgentRunner'. Or is there any better solution?

Reply:
Use a static block, i.e.:

```java
static {
...
}
```

Reply:
Unfortunately, the static code block priority is not as high as 'MethodAgentRule'.

Reply:
Hi @code98, you're right! I've [added a new feature in `AgentRunner`](https://github.com/opentracing-contrib/java-specialagent/blob/6c68b789365a158bc27db17b1fd6aa971de0c554/opentracing-specialagent/src/test/java/io/opentracing/contrib/specialagent/AgentRunner.java#L185-L192) to allow you to set system properties for a test. Please let me know if this works for you.

Reply:
good! I read the code you submitted and it works for me. Thank you!

Reply:
Hi @code98, will you remove this change to `AgentRunner`?

Reply:
yes, I will use your new feature 'properties'!

Reply:
Ok, please make all changes to this PR, and then I will approve it.